### PR TITLE
DCOS_OSS-722: Display Artifacts section in the Multi-container form

### PR DESCRIPTION
**Pull request by nLight**
The `More settings` section went missing during the simplification efforts. I extracted `Artifacts` as a component and added it back. 

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [x] Did you add new integration tests?
- [x] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/components/modals/NewCreateServiceModalForm.js [hunk 1]
```diff
@@ -333,6 +344,23 @@ class NewCreateServiceModalForm extends Component {
...
+
+          <AdvancedSection>
+            <AdvancedSectionLabel>
+              More Settings
+            </AdvancedSectionLabel>
+            <AdvancedSectionContent>
+              <h3 className="short-top short-bottom">
+                Advanced Settings
+              </h3>
+              <ArtifactsSection
```
Comment (wavesoft) [inline plugins/services/src/js/components/modals/NewCreateServiceModalForm.js]: Purely visual #nit ..  it kind of looks bad how the headings are stuck to each other. Can you ad an one-line description paragraph under the "Advanced Settings"? Something stupidly obvious, such as "Advanced options for the container" just to space things a bit 😄 

image

File: plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js [hunk 2]
```diff
@@ -5,16 +5,14 @@ import {Tooltip} from 'reactjs-components';
 import {FormReducer as ContainerReducer} from '../../reducers/serviceForm/Container';
 import {FormReducer as ContainersReducer} from '../../reducers/serviceForm/Containers';
 import {findNestedPropertyInObject} from '../../../../../../src/js/utils/Util';
-import AddButton from '../../../../../../src/js/components/form/AddButton';
+import ArtifactsSection from './ArtifactsSection';
```
Comment (wavesoft) [inline plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js]: Nice ✨

File: plugins/services/src/js/components/forms/ArtifactsSection.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+              Add Artifact
+            </AddButton>
+          </FormGroup>
+        </FormRow>
+      </div>
+    );
+  }
+}
+
+module.exports = ArtifactsSection;
```
Comment (wavesoft) [inline plugins/services/src/js/components/forms/ArtifactsSection.js]: `.propTypes` please 🚨

**Comment by orlandohohmeier** [comment]
Great Job! ✨  Feel free to merge once you've addressed @wavesoft's comments.

**Comment by nLight** [comment]
@wavesoft if you could verify and merge that would be amazing ✨

**Comment by MatApple** [comment]
Looks great 💯

**Pull request merged**